ffv1.md

```diff
@@ -233,15 +233,28 @@ RFC:```
 
 If the context is smaller than 0 then -context is used and the difference between the sample and its predicted value is encoded with a flipped sign.
 
-## Quantization
+## Quantization tables
 
-There are 5 quantization tables for the 5 sample differences, both the number of quantization steps and their distribution are stored in the bitstream. Each quantization table has exactly 256 entries, and the 8 least significant bits of the sample difference are used as index:
+Each quantization table contains 5 blocks, each block corresponding to 1 of the 5 sample differences, both the number of quantization steps and their distribution are stored in the bitstream. Each quantization table block has exactly 256 entries, and the 8 least significant bits of the sample difference are used as index:
```

Comment:
Except `each block corresponding to 1 of the 5 sample differences` comments (IMO order is explicated with `Q_1`, `Q_2`... And will be more explicit with the next PR changing them in an array), all comments are included in a the new version.